commit 2bdcbaed7d58da2eec73bd54c4914b0ca7ff9095 (refs/changes/04/1504/3)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2017-02-14T11:14:16-08:00 (2 years, 8 months ago)
    
    NAPI-346 NAPI's workflow code should be removed
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/docs/index.md b/docs/index.md
index 2f2d10f..26cb66e 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -10,7 +10,7 @@ markdown2extras: tables, code-friendly, fenced-code-blocks
 -->
 
 <!--
-    Copyright (c) 2015, Joyent, Inc.
+    Copyright (c) 2017, Joyent, Inc.
 -->
 
 # Networking API (NAPI)
@@ -348,12 +348,7 @@ This sets two static routes:
 
 ## UpdateNetwork (PUT /networks/:network_uuid)
 
-Updates a logical network.  Note updating the following parameters will cause
-a workflow to be run to update VMs on that network with the changes:
-
-* resolvers
-* routes
-* gateway
+Updates a logical network.
 
 ### Inputs
 
